doc/nodejs.rst

```diff
@@ -0,0 +1,80 @@
+Usage with Node.js
+==================
+
+Although remoteStorage.js was initially written for being used in browsers, we
+do support using it in a Node.js environment as well. See :doc:`this section
+</getting-started>` for getting started.
+
+The main difference between rs.js in a browser and using it on a server or in a
+CLI program is how to connect a storage. The RS protocol uses the OAuth
+Implicit Grant flow for clients to receive a bearer token, which they can use
+in HTTP requests.  This works by redirecting back to the Web application with
+the token attached to the redirect URI as a URI fragment.
+
+Now, with rs.js in a browser, calling
+``remoteStorage.connect('user@example.com')`` will take care of the entire
+OAuth process, including the parsing of the URI after the redirect, saving the
+token to localStorage and changing the library's state to connected. But in a
+node.js program, that's obviously not possible, because there's no browser that
+will open the OAuth dialog and receive the redirect with the token attached to
+the redirect URI.
+
+connect() with a token
+----------------------
+
+For this reason, among others, you can call the connect function with a token
+that you acquired beforehand::
+
+   remoteStorage.connect('user@example.com', 'abcdefghijklmnopqrstuvwxyz')
+
+This will skip the entire OAuth process, because you did that before in some
+other way, of course.
+
+Obtaining a token
+-----------------
+
+For some programs, like e.g. a server daemon, you can usually acquire the token
+from your server manually, and then just configure it for example as
+environment variable, when running your program.
+
+For CLI programs, and if you actually want to integrate the OAuth flow in your
+program, one possible solution is the following:
+
+1. Set up a simple Web site/app, which you publish under a fitting domain/URI
+   that you can use as the OAuth redirect URI.
+2. Have the user enter their user address and do a Webfinger lookup for auth
+   URL etc., e.g. using `webfinger.js
+   <https://www.npmjs.com/package/webfinger.js>`_.
+3. Create the OAuth request URI with the correct scope etc., and open a browser
+   window with that URI from your program.
```

Comment:
Maybe add "or prompt the user to open it".

Reply:
Makes sense. Thanks.